packages/conf-rust-2018/conf-rust-2018.1/opam

```diff
@@ -6,21 +6,25 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "MIT"
 extra-files: ["test.rs" "md5=639e04c270fef8589636e0416761a67b"]
```

Comment:
This change to conf-rust looks unrelated to lower bounds.

Reply:
I moved this into its own PR